munge/arpeng/run.js

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env node
+
+/*
+ * A launcher script for toimportjson.ts that sets up run-time transpilation.
+ * Written in JS for windows compatibility.
+ */
+
+const { spawn } = require("child_process");
+const { dirname, join } = require("path");
+
+const argv = process.argv.slice();
+const nodeExecutable = argv.shift();
+const thisScript = argv.shift();
+
+const targetScript = join(dirname(thisScript), "toimportjson.ts");
+
+let child;
+
+child = spawn(
```

Comment:
I looked into converting this into a _true_ `exec()` — that is, replacing the current process, but then realized that [doing that portably in Node is an unsolved problem](https://stackoverflow.com/q/51185115/6626414). So I guess this okay! 😅

Reply:
Yeah, windows doesn’t have exec :/ I’ll add a comment.